virttest/utils_stress.py

```diff
@@ -158,3 +163,27 @@ def vm_stress_events(self, event, vm):
                         libvirt.delete_local_disk(self.disk_type, disk_name)
             else:
                 raise NotImplementedError
+
+    def host_stress_event(self, event):
+        """
+        Host Stress events
+
+        :param event: event name
+        """
+        for itr in range(self.host_iterations):
+            if "cpufreqgovernor" in event:
```

Comment:
for better readability can we have the name like `cpu_freq_governor` for this and other events too.

Reply:
Sure, I can change it